src/tabset/tabset.ts

```diff
@@ -6,8 +6,10 @@ import {
   Directive,
   TemplateRef,
   ContentChild,
-  AfterContentChecked
+  AfterContentChecked,
+  Output
 } from '@angular/core';
+import {EventEmitter} from '@angular/common/src/facade/async';
```

Comment:
This should be imported from `@angular/core` as it is re-exported from there. Any path that has `src` is considered "private" and shouldn't be used.